### ISSUE TITLE
Can't copy dev scripts from docker

### ISSUE THREAD

**Issue by VadymTrofymenko** [sherifabdlnaby/kubephp#24]
**Describe the bug**
Got the error when trying to the run `make up` command

**To Reproduce**
1. Clone the repository
2. Run `make up`

![Screenshot from 2022-05-13 08-27-42](https://user-images.githubusercontent.com/40141016/168217039-cc96ff63-2956-4b48-b47b-35bd71800caa.png)

```
 > [app-dev 3/3] RUN chmod +x /usr/local/bin/*-dev;     mv "/usr/local/etc/php/php.ini-development" "/usr/local/etc/php/php.ini" COPY docker/php/dev-*   /usr/local/etc/php/conf.d/:                              
#23 0.381 mv: can't rename 'COPY': No such file or directory                                             
#23 0.381 mv: can't rename 'docker/php/dev-*': No such file or directory
------
executor failed running [/bin/ash -eo pipefail -c chmod +x /usr/local/bin/*-dev;     mv "$PHP_INI_DIR/php.ini-development" "$PHP_INI_DIR/php.ini" COPY docker/php/dev-*   $PHP_INI_DIR/conf.d/]: exit code: 1
ERROR: Service 'app' failed to build : Build failed
make: *** [Makefile:16: up] Error 1

```

Ubuntu 20.04.4
Docker 20.10.15

**Comment by sherifabdlnaby** [comment]
@VadymTrofymenko Hello, thank you for spotting this out. It was addressed in draft v5.0 version of kubephp.

**Comment by sherifabdlnaby** [comment]
@VadymTrofymenko Thanks for opening a PR, while this is fixed for v5.0, we'll need a patch for <= v4.0^. I reviewed the changes and have requested some changes. ✌🏻

**Comment by VadymTrofymenko** [comment]
> @VadymTrofymenko Thanks for opening a PR, while this is fixed for v5.0, we'll need a patch for <= v4.0^. I reviewed the changes and have requested some changes. ✌🏻

Got it, Thanks!

**Comment by sherifabdlnaby** [comment]
@VadymTrofymenko Thank you!
